openapi/.redocly.yaml

```diff
@@ -0,0 +1,71 @@
+# See https://docs.redoc.ly/cli/configuration/ for more information.
+
+organization: adobe
+extends:
+    - minimal
+apis:
+  Commerce(Admin)@2.4.5:
+    root: openapi/admin-schema-2.4.5.yaml
+    labels:
+      - main
+  Commerce(Customer)@2.4.5:
+    root: openapi/customer-schema-2.4.5.yaml
+  Commerce(Guest)@2.4.5:
+    root: openapi/guest-schema-2.4.5.yaml
+  Commerce(Admin)@2.3.7:
+    root: openapi/admin-schema-2.3.7.yaml
+  Commerce(Customer)@2.3.7:
+    root: openapi/customer-schema-2.3.7.yaml
+  Commerce(Guest)@2.3.7:
+    root: openapi/guest-schema-2.3.7.yaml
+features.openapi:
+  pagination: section
+  hideHostname: true
+  sortPropsAlphabetically: true
+  pathInMiddlePanel: true
+  hideDownloadButton: false
+  hideTryItPanel: true
+  showExtensions: true
+  nativeScrollbars: true
+  ctrlFHijack: true
+  scrollYOffset: .site-nav
+  theme:
```

Comment:
I'm not sure if this `theme` section is necessary anymore. You may want to ask @timkim, but I imagine that we'll want all Adobe API docs (not just commerce) to have the same look and feel. This theme section might compromise the consistency across non-Commerce API docs.